spaces/SouthChain/index.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "SouthChain",
+  "network": "100", 
+  "symbol": "SCT",  
+  "skin": "southchain.scss", 
```

Please delete scss extension to keep only the selector.
```suggestion
  "skin": "southchain", 
```